src/iframe-helper.js

```diff
@@ -190,8 +190,9 @@ function dropListenSentinel(listenSentinel) {
 /**
  * Registers the global listenFor event listener if it has yet to be.
  * @param {!Window} parentWin
+ * @param {string=} typeKey Key to index event type.
  */
-function registerGlobalListenerIfNeeded(parentWin) {
+function registerGlobalListenerIfNeeded(parentWin, typeKey = 'type') {
   if (parentWin.listeningFors) {
```

Comment:
Are you sure this will perform the way you expect?  My fear is that the first to call will impose what typeKey is required.

Reply:
You're right. Reverting to something similar to what I had in my first approach. I'll take an optional typeKey, but check the default 'type' as well.

Reply:
Alternatively, we can maintain a list (either module-level or in global object) of type keys and check against those.